draft-ietf-rats-eat.md

```diff
@@ -1099,12 +1103,53 @@ string naming the submodule. No submodules may have the same name.
 {::include cddl/submods.cddl}
 ~~~~
 
+
 # Endorsements and Verification Keys {#keyid}
 
 TODO: fill this section in. It will discuss key IDs, endorsement ID and such that
 are needed as input needed to by the Verifier to verify the signature. This will
 NOT discuss the contents of an Endorsement, just and ID/locator.
 
+
+# Layered Attestation {#layered}
+
+It is possible for a device to have multiple Attesting Environments as described in {{RATS-Architecture}}.
+When these are layered, one is a root the endorsement and/or verification key originates with the device manufacturer or endorser.
+The trust for those that are layered above the root comes from Attestation Evidence about them that is provided by the root or a layer below.
```

Comment:
This addresses the concern above, but later in the text. Maybe some minor rewording above suffices.